src/views/HomeView.vue

```diff
@@ -1,10 +1,222 @@
+
+      const [ teamOneSpreadId, teamOneMLId ] = teamOneData.betIds
+      const teamOneSpread = bets.value?.find(bet => bet.id === teamOneSpreadId).lineage
+      const teamOneML = bets.value?.find(bet => bet.id === teamOneMLId).lineage
+
+      const [ teamTwoSpreadId, teamTwoMLId ] = teamTwoData.betIds
+      const teamTwoSpread = bets.value?.find(bet => bet.id === teamTwoSpreadId).lineage
+      const teamTwoML = bets.value?.find(bet => bet.id === teamTwoMLId).lineage
+      
+      return {  
+        id: eventId,
+        firstTeam: {
+          name: teamOneInfo.name,
+          id: teamOneInfo.id,
+          spread: { id: teamOneSpreadId, 
+                    value: teamOneSpread },
+          ml: { id: teamOneMLId,
+                value: teamOneML } 
+        },
+        secondTeam: {
+          name: teamTwoInfo.name,
+          id: teamTwoInfo.id,
+          spread: { id: teamTwoSpreadId, 
+                    value: teamTwoSpread },
+          ml: { id: teamTwoMLId,
+                value: teamTwoML } 
+          },
+      }
+    })
+
+    return {
+      name: sport.name,
+      initial: sport.key,
+      allEvents: sportEvents
+    }
+  })
+})
+
+const getIcon = sportKey => {
+  switch (sportKey) {
+    case 'nhl':
+      return NhlIcon;
+    case 'mlb':
+      return MlbIcon;
+    case 'nfl':
+      return NflIcon;
+    case 'nba':
+      return NbaIcon;
+  }
+}
 
+const sportsWithIcon = computed(() => allSports.value.map(sport => ({
+  ...sport,
+  initials: sport.key.toUpperCase(),
+  icon: getIcon(sport.key)
+})))
+
+// // Submit button will have: 'eventId', 'betId', 'teamId'
+function sendBet(eventID, teamID, betObj){
+const bet = {
+        betId: betObj.id,
+        eventId: eventID,
+        teamId: teamID };
+
+axios.post('/submit', bet)
+  .then((result) => (console.log(result.data)))
+}
+</script>
 <template>
-  Homepage
+  <div class="container">
+    <Header>
+      <HeaderItem
+        v-for="sport in sportsWithIcon"
+        :key="sport.key"
+        :initials="sport.initials"
+        :icon="sport.icon"/>
+    </Header>
+
+    <aside class="sidebar">
+      Sidebar
+    </aside>
+
+<!-- TODO: Figure out how to make this less verbose -->
+    <main class="content">
+      <div v-for="sport in eventRows"
+      :key="sport.initial" 
+      class="sportRow">
+        <h2>{{ sport.name }}</h2>
+        <div class="cards">
+          <EventCard v-for="event in sport.allEvents"
+          :key="event.id" 
+          class="card">
+            <template #eventTitle>
+                <h3>{{ event.firstTeam.name }} vs {{ event.secondTeam.name}}</h3> 
+            </template>
+            <template #teamOneName>
+              {{ event.firstTeam.name }}
+            </template>
+            <template #teamOneML>
+              <Button
+                @click="sendBet(event.id, event.firstTeam.id, event.firstTeam.ml)">
+                <template #value>
```

Comment:
Since you're passing the id for event and firstTeam, you can also do the same for ml, which means:

`sendBet(event.id, event.firstTeam.id, event.firstTeam.ml.id)`

Then you don't have to break it down in the function above.

Also, only break attr's across a line if there's more than 2 on a DOM tag (even though I know that one is super long).